### DISCUSSION TITLE
Still getting "Unreachable DNS" error by Google crawlers

### DISCUSSION THREAD

**Discussion by URubino4Geeks** [vercel/community#2664] [Help]
### Summary

Regarding the Incident Report linked below, I'm still getting DNS error messages from Google services regarding "Unreachable DNS". The incident was resolved 5 days ago, is this related to that incident or might be another issue?

https://www.vercel-status.com/incidents/8w42qmnv83cj

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/129084465/bc0b01fb-ebf7-4c23-b7d8-bd32eabcb9ac)

**Comment by smaeda-ks** [comment]
let's move this to https://github.com/orgs/vercel/discussions/2611#discussioncomment-6047511
